app/models/project.rb

```diff
@@ -15,5 +15,15 @@ class Project < ApplicationRecord
   has_one  :owner, through: :owner_membership, source: :user
   has_many :contributors, through: :contributor_memberships, source: :user
 
+  has_many :resources, class_name: 'ProjectResource'
+  has_many :requests,
```

Comment:
What do you think about the deviation of naming here?

I'm wondering whether we should do:

```ruby
has_many :info_requests
# …
has_many :info_request_batches
#…
```

And then to get all requests…

```ruby
def all_info_requests
  # …
```

I'm in two minds. On minus side for the current implementation, we have to remember `requests` and `batches` which is unusual. On the plus side, we _don't_ have to remember that there's a gotcha when you just want to list all the info requests, irrespective of how they were made.

I don't think I feel strongly either way, so I guess my default is to say keep it as is, but just wanted to pose the question to see what you think.

Reply:
No strong feelings on this. I think we go for what is there now and see how it works. Easy to change if we find it confusing.